examples/jsm/exporters/USDZExporter.js

```diff
@@ -36,7 +36,7 @@ class USDZExporter {
 					materials[ material.uuid ] = material;
 					if ( material.map !== null ) textures[ material.map.uuid ] = material.map;
 					if ( material.normalMap !== null ) textures[ material.normalMap.uuid ] = material.normalMap;
-					if ( material.aoMap !== null ) textures[ material.aoMap.uuid ] = material.aoMap;
+					//if ( material.aoMap !== null ) textures[ material.aoMap.uuid ] = material.aoMap;
```

Comment:
For now I skip AO because it is broken when using Transform2D. For some reason it is always scaled the same amount as albedo (diffuse) map, for example if albedo has scale (8,8) and you obviously want your AO map to remain (1,1), no luck here, AO will be also (8,8).